concepts/05 Widgets/CheckBox/10 Handle the Value Change Event.md

```diff
@@ -18,14 +18,14 @@ To process a new **CheckBox** value, you need to handle the value change event.
 
     <!--HTML-->
     <dx-check-box
-        (onValueChanged)="checkBox_valueChanged($event)">
+        (onValueChanged)="handleValueChange($event)">
```

Comment:
Let's add two-way binding to the example.

Reply:
fixed